src/components/menu/menu-item/menu-item.component.tsx

```diff
@@ -242,6 +242,7 @@ export const MenuItem = ({
     rel,
     onClick,
     icon,
+    hideAriaLabel: true,
```

Comment:
**suggestion**: (this isn't about this line of code in particular but seems the best place I can find to put it)

I think it's worth adding a test for MenuItem specifically, that we don't end up with an aria-label on the StyledIcon when rendering an icon. I know you have the test for the new prop on Icon itself, and that will satisfy coverage, but you're nowhere testing that we're actually applying this in MenuItem, and that was after all what the ticket is really about.

Reply:
The reason I tested it on `Icon` was that it wasn't technically `MenuItem` that had the bug. The bug came from `Link` which in turn came from `Icon` getting a aria-label set. I'll get a test added to `MenuItem` though as it's not that big of a job to do for the extra piece of mind it can give us 👍🏻

Reply:
Actually you're right that it's Link where the bug actually was, so a test on that is not a bad idea. I'd definitely prefer to have one on either that or Menuitem (or even both, but probably no need), than neither, though 👍

Reply:
I've amended one on `MenuItem` and added one to `Link` based on the changes made in the last commit. So we're covered for both now.

`MenuItem` checks `Link` and `Icon`.
`Link` checks `Link` and `Icon`.

So if a regression happens in `Link` or `MenuItem`, we'll know about it 👍🏻